logging_config.py

```
@@ -0,0 +1,497 @@
+                    'request_id': request_id,
+                    'correlation_id': correlation_id
+                }
+            )
+            
+            return result
+            
+        except Exception as e:
+            duration_ms = (time.time() - start_time) * 1000
+            
+            # Log error
+            logger.error(
```

Ruff (0.12.2)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)